website/ibm.erb

```diff
@@ -540,6 +540,9 @@
             <li<%= sidebar_current("docs-ibm-resource-dns-record") %>>
               <a href="/docs/providers/ibm/r/dns_record.html">dns_record</a>
             </li>
+            <li<%= sidebar_current("docs-ibm-resource-dns-zone") %>>
```

Comment:
Declare complete new section Private DNS resource like object storage and move to that section